src/components/dialogs/parameters/shortcircuit/short-circuit-parameters.tsx

```diff
@@ -197,9 +205,12 @@ const ShortCircuitFields: FunctionComponent<ShortCircuitFieldsProps> = ({
         isIccMinFeaturesDefaultConfiguration,
     ]);
 
-    // reset all fields when predefined parameters changes
+    // Reset all fields when predefined parameters change; otherwise, save configured parameters.
     useEffect(() => {
-        resetAll(watchPredefinedParams);
+        if (predefinedParamsRef.current !== watchPredefinedParams) {
```

Comment:
```suggestion
        // compare with ref to prevent executing resetAll on first render
        if (predefinedParamsRef.current !== watchPredefinedParams) {
```